pkg/http-server/start.go

```diff
@@ -61,14 +62,24 @@ func (g *APIServer) start(routes []*Route, port string) {
 		router.Methods(v.verb).Path(v.path).HandlerFunc(v.fn)
 	}
 
+	// Add a route for all static templates / assets. Currently used for the Webhook logs views
+	// go/terrascan/asset is the path where the assets files are located inside the docker container
+	router.PathPrefix("/assets/").Handler(http.StripPrefix("/assets/", http.FileServer(http.Dir("/go/terrascan/assets"))))
```

Comment:
hardcoding `/go/terrascan/assets` can cause potential bugs. maybe we can introduce an env variable for this directory path and keep this value as default.

Reply:
Introducing a new environment variable may not be the best way to go ahead. This path is not something that an end user would be configuring. May be we can document that this endpoint would work on docker containers